source/ed/ed2nav.cpp

```diff
@@ -168,6 +169,10 @@ int main(int argc, char * argv[])
         ("min_non_connected_ratio,m",
          po::value<double>(&min_non_connected_graph_ratio)->default_value(0.1),
          "min ratio for the size of non connected graph")
+        ("full_street_network_geometries", po::value<bool>(&export_georef_edges_geometries)->default_value(false),
```

Comment:
it's minor but you don't have to define a type, it's more a flag that we want to activate, without a type we can do this:
```
ed2nav --full_street_network_geometries <blabla>
```
then you use ``` vm.count("full_street_network_geometries") ``` for setting the value of ```export_georef_edges_geometries```